### PR TITLE
React to deprecation of Fedora.28.Amd64.Open

### PR DESCRIPTION
**Pull request by natemcmaster**
The dedicated Fedora.28.Amd64.Open queue will be shutdown on May 29. This updates to use the Docker queue instead.

### CODE REVIEW

**Comment by natemcmaster** [comment]
Getting a consistent failures now in Fedora tests.

> mkdir: cannot create directory '/root/helix/work/correlation/sdk': Read-only file system

Must be missing something obvious. @MattGal @HaoK is the entire filesystem read-only when using dockerized Helix? Or just some directories?

**Comment by HaoK** [comment]
Yeah we can't write to the correlation payloads on the dockerized queues, we will need to switch to a different directory for these queues

**Comment by HaoK** [comment]
Alternatively, maybe we should just pull upload the cli/sdk as correlation payloads instead of having each worker install them?

File: eng/targets/Helix.Common.props [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <HelixAvailableTargetQueue Include="Debian.8.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Debian.9.Amd64.Open" Platform="Linux" />
     <HelixAvailableTargetQueue Include="Redhat.7.Amd64.Open" Platform="Linux" />
-    <HelixAvailableTargetQueue Include="Fedora.28.Amd64.Open" Platform="Linux" />
+    <HelixAvailableTargetQueue Include="(Fedora.28.Amd64.Open)Ubuntu.1604.Amd64.Open@mcr.microsoft.com/dotnet-buildtools/prereqs:fedora-28-helix-09ca40b-20190508143249" Platform="Linux" />
```
Comment (HaoK) [inline eng/targets/Helix.Common.props]: What does the paren syntax mean?

Comment (MattGal) [reply eng/targets/Helix.Common.props]: The parenthesis part is queue spoofing for reporting purposes only; if you don't need it, you can leave it out.  If the run is viewed on Mission Control without this, it'd report as the machines it actually ran on, Ubuntu.1604.Amd64.Open, which is confusing once you're using a bunch of them.

**Comment by natemcmaster** [comment]
Updated to put the SDK into the working directory instead of the correlation folder.

Looks like test failures now are due to the normal flakiness. Going to retry once more and then will add Flaky attributes.

**Comment by dougbu** [comment]
Lots of very consistent Blazor test failures plus numerous random-seeming test failures. Error for the Blazor tests is:
```
System.IO.FileNotFoundException : Could not find file '/home/helixbot/work/309432f4-a992-4f9f-bafa-2a93fbf1a09c/Work/636c0f27-9b3f-4806-9849-6df7f6a3e897/Exec/referenceHints.txt'
```

In addition, the test view does not show anything for Fedora ☹️

**Comment by MattGal** [comment]
@dougbu not sure what's up with the Fedora but the MC view from the logs (https://mc.dot.net/#/user/aspnetcore/pr~2Faspnet~2Faspnetcore/ci/20190521.2 ) shows 100% pass rate for Fedora 28 so that's good.

**Comment by natemcmaster** [comment]
112 failures overall. Going to take a closer look and start applying `[Flaky]` attributes.

FYI @aspnet/build

**Comment by natemcmaster** [comment]
Upon further review, it appears I broke the existing skip conditions. I fixed up the environment variables we are setting and added a comment to code to indicate that the HELIX env var is expected to be set to the 'friendly' queue name (not the fully-qualified version which lists the docker image)

**Comment by natemcmaster** [comment]
Only test failure on Helix now is https://github.com/aspnet/AspNetCore-Internal/issues/2490. 

I'd like to merge this today. @HaoK can you review and sign off if the changes look ok?

**Review by HaoK — APPROVED** [review]
Nice, this will helpful in the arm PR too

**Pull request merged**